src/Raven.Server/Documents/TimeSeries/TimeSeriesPolicyRunner.cs

```diff
@@ -61,12 +64,19 @@ public static TimeSeriesPolicyRunner LoadConfigurations(DocumentDatabase databas
             }
             catch (Exception e)
             {
-                
                 const string msg = "Cannot enable policy runner as the configuration record is not valid.";
-                database.NotificationCenter.Add(AlertRaised.Create(
-                    database.Name,
-                    $"policy runner error in {database.Name}", msg,
-                    AlertType.RevisionsConfigurationNotValid, NotificationSeverity.Error, database.Name));
+
+                if (e is LicenseLimitException lle)
+                {
+                    LicenseLimitWarning.AddLicenseLimitNotification(database.ServerStore.NotificationCenter, lle);
+                }
+                else
+                {
+                    database.NotificationCenter.Add(AlertRaised.Create(
+                        database.Name,
+                        $"policy runner error in {database.Name}", msg,
```

Comment:
Why the title starts from lower letter here? Also it is hard to know from the title what policy runner it is?

Reply:
fixed the latter.
There only one runner per database